packages/yarnpkg-core/sources/Workspace.ts

```diff
@@ -69,7 +69,7 @@ export class Workspace {
         expandDirectories: false,
         onlyDirectories: true,
         onlyFiles: false,
-        ignore: [`node_modules`, `.git`],
+        ignore: [`**/node_modules/**`, `**/.git/**`, `**/.yarn/**`],
```

Comment:
Shouldn't it be `**/node_modules` (and similar)? The doc isn't very clear, but:

> ⚠️ When you write `!**/second/**/*` it means that the directory will be read, but all the entries will not be included in the results.

Assuming they made a mistake and aimed to write `!**/second/**`, then we would want to just ignore iterating inside those folders, for perf reasons, right?

Reply:
@arcanis Yep, you are right, good catch! Updated ignore patterns